### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by Rchardon**

### CODE REVIEW

**Comment by Zamiell** [comment]
what does randomBaby equal if babies mod is not enabled?

**Comment by Rchardon** [comment]
Nil maybe ? But does that matter ? If the babies mod is not enabled then the statement is false and that should be ok

**Comment by Rchardon** [comment]
I checked the value of randomBaby when the mod isn't activated, it's -1

File: src/features/optional/quality/openAntibirthDoor.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Automatically open Antibirth doors in races to Mother
+
+import g from "../../../globals";
+
+export function main(): void {
+  const player = Isaac.GetPlayer();
+  if (g.r.GetType() === RoomType.ROOM_BOSS && g.r.IsClear()) {
+    for (let i = 0; i < 8; i++) {
+      const Door = g.r.GetDoor(i);
```
Comment (Zamiell) [inline src/features/optional/quality/openAntibirthDoor.ts:9]: don't capitalize door

Comment (Zamiell) [reply src/features/optional/quality/openAntibirthDoor.ts:9]: also use the getdoors function instead

**Pull request closed without merging**